commands/create-account.js

```diff
@@ -4,8 +4,8 @@ const connect = require('../utils/connect');
 const { KeyPair } = require('near-api-js');
 const eventtracking = require('../utils/eventtracking');
 
-module.exports = {
-    command: 'create_account <accountId>',
+const createAccountCommand = {
+    command: 'create-account <accountId>',
     desc: 'create a new developer account',
     builder: (yargs) => yargs
         .option('accountId', {
```

Comment:
Don't we want to change these options to use dash-case also? I think `yargs` will accept them in either format, but I think we should probably show them in dash-case in any help commands.

Reply:
fixing options is a larger undertaking which has some risk (see https://github.com/near/near-shell/pull/414), so I'd prefer to do it as a separate effort.

Reply:
Yeah, I wondered if that would be more work than it's worth right now. I think the change from `create_account` to `create-account` is important enough that we can leave the options as-is for now.